src/populate.js

```diff
@@ -0,0 +1,14 @@
+// get scores from API
+
+export default async function populate() {
+  const requestOptions = {
+    method: 'GET',
+    redirect: 'follow',
+  };
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/dV4aWISG7S2Gi5e0alOq/scores';
+  const request = new Request(requestURL);
+  const response = await fetch(request, requestOptions);
+  const superHeroesText = await response.text();
+  const superHeroes = JSON.parse(superHeroesText);
+  return superHeroes;
+}
```

Comment:
- [ ] Well done writing this function and it's all working fine but the issue is that one of the requirements in this milestone is to use ES6 arrow functions instead of the regular `function` keyword. so kindly consider refactoring it.
project requirement
  -  Use arrow functions instead of the function keyword.

Reply:
Thanks, I thought It was not mandatory for all functions inside the project.